page-eight-two.php

```diff
@@ -0,0 +1,225 @@
+		<div class="row headerrow">
+			<div class="col-md-5">
+				<iframe width="95%" height="315" src="https://www.youtube.com/embed/RrAhClhrRAc?rel=0" frameborder="0" style="margin-top:10px;" allowfullscreen></iframe>
+			</div>
+			<div class="col-md-7">
+				<h2>ownCloud Server 8.2 Features</h2>
+				<p>ownCloud Server 8.2 has significant improvements for users, administrators and developers.
+					This page contains an overview of the most important changes.
+				</p>
+				<a href="/install" class="btn btn-lg btn-primary install-8-btn">Install ownCloud 8.2 now</br></a>
+				<p>Want to learn more about ownCloud from somebody in person? Meet us at an event or join the conversation online!</p>
+				<div class="evnt-btn-wrap">
+					<a href="/events" class="btn btn-lg btn-default">Join an ownCloud event</a>
+					<a href="/promote" class="btn btn-lg btn-default">Discuss ownCloud with others</a>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+<!-- end main -->
+
+<div class="feature-wide-row-wrap">
+	<div class="wrap container">
+		<div class="row">
+		 	<div class="col-md-4 col-md-offset-4">
+		  		  <h3 class="row-center-title">Release highlights</h3>
+		  	</div>
+		 </div>
+			<div class="col-md-4 featureblock">
+				<i class="icon-desktop"></i>
+				<h3 class="ft-title">Improved Design</h3>
+				<p class="featuredesc">This release introduces significant design improvements including new icons and a sidebar giving quick access to file information.</p>
+			</div>
+
+			<div class="col-md-4 featureblock">
+				<i class="icon-picture-o"></i>
+				<h3 class="ft-title">Pictures Becomes Gallery</h3>
+				<p class="featuredesc">The Gallery app replaces the Pictures with a much nicer user experience introducing image sorting, image zoom, easy switching to the gallery and more.</p>
```

Comment:
Call it the old app "the Pictures app" - consistent with what you already did further down.